commands/quotes.py

```diff
@@ -1,124 +1,108 @@
-    if not is_user_id_linked_to_verified_user(ctx.guild.id, ctx.author.id):
-        replyEmbed = create("AddQuote Reply", "Invalid Permissions", "red")
+    if not _mongoFunctions.is_user_id_linked_to_verified_user(ctx.guild.id, ctx.author.id):
+        replyEmbed = _embedMessage.create("AddQuote Reply", "Invalid Permissions", "red")
         await ctx.channel.send(embed = replyEmbed)
         return
 
-    args = parse_message(ctx.content)
+    args = _util.parse_message(ctx.content)
 
     if len(args) != 3:
-        await ctx.channel.send(embed = create("AddQuote Reply", "Invalid Syntax! You need two arguments for this function!", "red"))
+        await ctx.channel.send(embed = _embedMessage.create("AddQuote Reply", "Invalid Syntax! You need two arguments for this function!", "red"))
         return
 
     if len(args[1]) > 1024:
-        await ctx.channel.send(embed = create("AddQuote Reply", "Quote is too long! Please submit a quote that is 1024 characters or fewer", "red"))
+        await ctx.channel.send(embed = _embedMessage.create("AddQuote Reply", "Quote is too long! Please submit a quote that is 1024 characters or fewer", "red"))
         return
 
-    embed = create("AddQuote Reply", "| \"" + args[1] + "\" by: " + args[2] + " submitted by: " + ctx.author.mention + " \n Approved by: ", "blue")
+    embed = _embedMessage.create("AddQuote Reply", "| \"" + args[1] + "\" by: " + args[2] + " submitted by: " + ctx.author.mention + " \n Approved by: ", "blue")
     message = await ctx.channel.send(embed = embed)
-    await message.add_reaction(discord.utils.get(ctx.guild.emojis, name = "bedi"))
-
-    # await ctx.channel.send("Quote Recorded!")
+    await message.add_reaction(discord.utils.get(ctx.guild.emojis, name = _mongoFunctions.get_settings(ctx.guild.id)['reaction_emoji']))
 
 
 async def get_quotes(ctx: discord.message, client: discord.client):
-    args = parse_message(ctx.content)
+    args = _util.parse_message(ctx.content)
     if len(args) != 3 and len(args) != 2:
-        await ctx.channel.send(embed = create("getQuote Reply", "Invalid Syntax! You need two arguments for this function!\nEx: $getQuotes Bedi 2", "red"))
+        await ctx.channel.send(embed = _embedMessage.create("getQuote Reply", "Invalid Syntax! You need two arguments for this function!\nEx: $getQuotes Bedi 2", "red"))
         return
     try:
         person = str(args[1])
         if len(args) == 2:
             page = 1
         else:
             page = int(args[2])
-        quotes = find_quotes(ctx.guild.id, person, page)
+        quotes = _mongoFunctions.find_quotes(ctx.guild.id, person, page)
         try:
-            # print(quotes)
-            embed = create("Quotes from: " + person, "Page: " + str(page), "green")
+            embed = _embedMessage.create("Quotes from: " + person, "Page: " + str(page), "green")
             for quote in quotes:
                 value = (quote["quote"][:1021] + '...') if len(quote["quote"]) > 1024 else quote["quote"]
```

Comment:
This line should have a comment as we might get confused with why we chose 1021 and 1024 lol

Reply:
Instead I created a variable to represent 1024 and when 1021 was used, I used "1024 - 3"